src/components/buttonsIndicators/button/style.ts

```diff
@@ -116,7 +117,7 @@ export const StyledText = styled<Props, 'div'>('div')`
   align-items: center;
   text-align: center;
   letter-spacing: 0;
-  font-weight: 500;
+  font-weight: ${p => p.heavy ? '700' : '500'};
```

Comment:
@jenn-rhim @karenkliu what semantically makes a Button get heavy text? Should all Primary action buttons be heavy for example? If it's more arbitrary than that, can we get a 1-sentence rule describing the reason it's used for this 1 instance here and perhaps something semantic will come out of that?